frontends/web/package.json

```diff
@@ -16,14 +17,19 @@
     "build/*"
   ],
   "devDependencies": {
+    "@types/jest": "^23.3.2",
```

Comment:
Again not my area of expertise but I guess/thought we want to fix the versions of our dependencies by removing the `^`.

Reply:
And don't forget to also commit an updated yarn.lock then.

Reply:
done